saleor/app/installation_utils.py

```diff
@@ -269,6 +269,7 @@ def install_app(app_installation: AppInstallation, activate: bool = False):
             url=extension_data.get("url"),
             mount=extension_data.get("mount"),
             target=extension_data.get("target", AppExtensionTarget.POPUP),
+            # TODO Drop in 3.23
```

Comment:
TODOs like this one stays in the code forever 😅  Make sure that you created the task and assign it to the 3.23 release

Reply:
Hhm, btw why TODO instead of dropping it? current `main` is actually 3.23 🤔